webpack/assets/javascripts/react_app/components/notifications/drawer/NotificationDropdown.js

```diff
@@ -2,24 +2,22 @@ import React from 'react';
 import { Dropdown, Glyphicon, MenuItem } from 'react-bootstrap';
 
 const NotificationDropdown = ({ links, id }) => {
-  const listLinks = links.map((link, i) => {
-    const liKey = `notification-link-${i}-${id}`;
-
-    return (
-      <MenuItem key={liKey} id={liKey} target="_blank" href={link.href}>
-        {link.title}
-      </MenuItem>
-    );
-  });
-
   return (
-    <Dropdown className="pull-right dropdown-kebab-pf" pullRight key={id}
-              id={`notifications-dropdown-${id}`}>
+    <Dropdown pullRight className="dropdown-kebab-pf" id={id}>
       <Dropdown.Toggle noCaret bsStyle="link">
         <Glyphicon bsClass="fa" glyph="ellipsis-v" />
       </Dropdown.Toggle>
       <Dropdown.Menu>
-        {listLinks}
+        {
+          links.map((link, i) =>
+            <MenuItem key={i}
+                      id={ `notification-kebab-${i}` }
+                      target={link.external ? '_blank' : '_self'}
```

Comment:
please ensure that once the target is self, then close the notification drawer when clicking on an action.